solidity/test/modules/multisigModule.test.ts

```diff
@@ -0,0 +1,328 @@
+/* eslint-disable @typescript-eslint/no-floating-promises */
```

Comment:
This might just be me but a folder called `modules` feels to generic even though for most folks its maybe clear that ISMs are meant here?